pkgs/by-name/pl/plasticscm-theme/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  dpkg,
+  fetchurl,
+  lib,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "plasticscm-theme";
+  version = "11.0.16.9656";
+
+  src = fetchurl {
+    url = "https://www.plasticscm.com/plasticrepo/stable/debian/amd64/plasticscm-theme_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-GSZ2j/xZOUDI2SoqLWeGYq8YdPGLJ7ZXaJaOrsru8PY=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+  ];
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
+  dontFixup = true;
```

Comment:
Please do not disable phases which just run and do not cause issues. As we otherwise just make overrides hard.